content/posts/20220928-Innovation-22.md

```diff
@@ -0,0 +1,12 @@
+---
+title: "OPI Project at Intel Innovation 2022"
+date: 2022-09-28
+---
+
+At the recent [Intel Innovation 22](https://www.intel.com/content/www/us/en/events/on-event-series/innovation.html) event, a presentation on
+the Open Programmable Infrastructure Project was presented by Dan Daly
+(Intel) and Paul Pindell (F5). See below for the link to the recording.
+
+[Open Programmable Infrastructure (OPI) Using IPDK](https://www.intel.com/content/www/us/en/events/on365/open-programmable-infrastructure-using-ipdk.html)
+
+You can view the slides [here](/presentations/IntelON-OPI-IPDK.pdf)
```

Comment:
Small nit: Can you add a period (`.`) after the link?

Reply:
sure. Just to be sure I'm not missing a Git syntax thing... The request is simply a grammar thing -- missing a period at the end of a sentence.

Reply:
Yup, just a grammar thing! Thanks for the fix!